Source/ModelGroups/Node_Attribute.cs

```diff
@@ -1,167 +1,136 @@
+                    Properties = properties,
+                    GLTF = CreateGLTF(() => gltf.Scenes[0]),
+                };
+            }
+
+            void SetTranslation(List<Property> properties, Runtime.Node node)
             {
-                node.Mesh.MeshPrimitives[0].Normals = null;
-                node.Mesh.MeshPrimitives[0].Tangents = null;
+                node.Translation = new Vector3(-2, 2, -2);
+                properties.Add(new Property(PropertyName.Translation, node.Translation));
             }
 
-            // Texture the model
-            foreach (Property req in requiredProperty)
+            void SetTranslationX(List<Property> properties, Runtime.Node node)
             {
-                if (req.name == Propertyname.BaseColorTexture)
-                {
-                    material.MetallicRoughnessMaterial = new Runtime.PbrMetallicRoughness();
-                    material.MetallicRoughnessMaterial.BaseColorTexture = new Runtime.Texture();
-                    material.MetallicRoughnessMaterial.BaseColorTexture.Source = req.value;
-                }
+                node.Translation = new Vector3(-2, 0, 0);
+                properties.Add(new Property(PropertyName.Translation, node.Translation));
             }
 
-            foreach (Property property in combo)
+            void SetTranslationY(List<Property> properties, Runtime.Node node)
             {
-                if (property.name == Propertyname.Matrix)
-                {
-                    nodeList[1].Matrix = specialProperties[0].value;
-                }
-                else if (property.name == Propertyname.Translation ||
-                         property.name == Propertyname.Translation_X ||
-                         property.name == Propertyname.Translation_Y ||
-                         property.name == Propertyname.Translation_Z)
-                {
-                    nodeList[1].Translation = property.value;
-                }
-                else if (property.name == Propertyname.Rotation)
-                {
-                    nodeList[1].Rotation = specialProperties[1].value;
-                }
-                else if (property.name == Propertyname.Scale)
-                {
-                    nodeList[1].Scale = property.value;
-                }
+                node.Translation = new Vector3(0, 2, 0);
+                properties.Add(new Property(PropertyName.Translation, node.Translation));
+            }
+
+            void SetTranslationZ(List<Property> properties, Runtime.Node node)
+            {
+                node.Translation = new Vector3(0, 0, -2);
+                properties.Add(new Property(PropertyName.Translation, node.Translation));
+            }
+
+            void SetRotation(List<Property> properties, Runtime.Node node)
+            {
+                var rotation = Quaternion.CreateFromAxisAngle(new Vector3(0f, 1f, 0f), (float)Math.PI);
+                rotation.W = (float)Math.Round(rotation.W);
```

Comment:
huh?

Reply:
This has been removed, but the model will be shown as having been changed.
Since we're using a float, we get a value that is nearly but not quite zero. I added this in an effort to keep the number used for the rotation the same as what we're displaying, but it really doesn't matter.

Reply:
Maybe you can just set the Quaternion explicitly. `new Quaternion(0.0f, 1.0f, 0.0f, 0.0f)`

Reply:
That worked. Fixed